doc/source/dev/core-dev/github.rst.inc

```diff
@@ -117,5 +117,5 @@ which is about Trac (what we used pre-GitHub) tickets.
 *PR naming convention:* Pull requests, issues and commit messages usually start
 with a three-letter abbreviation like ``ENH:`` or ``BUG:``.  This is useful to
 quickly see what the nature of the commit/PR/issue is.  For the full list of
-abbreviations, see `writing the commit message
-<https://docs.scipy.org/doc/numpy/dev/gitwash/development_workflow.html#writing-the-commit-message>`_.
+abbreviations, see
+:std:label:`dev/development_workflow.html#writing-the-commit-message`.
```

Comment:
:std:label: is not known to the sphinx version used

Reply:
I think it should be fixed now using `:ref:` for cross-referencing,  let's see if the docs build fine. The previous link was broken and the label was found from `python -msphinx.ext.intersphinx https://docs.scipy.org/doc/numpy/objects.inv`.